tpm/tpm_test.go

```diff
@@ -74,3 +76,9 @@ func Test_close(t *testing.T) {
 	closeTPM(context.Background(), newCloseErrorTPM(t), &closeErr)
 	require.EqualError(t, closeErr, "failed closing attest.TPM: closeErr") // attest.TPM is backed by the closeSimulator
 }
+
+func TestTPMNoStorageConfiguredError(t *testing.T) {
+	err := ErrNoStorageConfigured
+	require.ErrorIs(t, err, ErrNoStorageConfigured)
```

Comment:
This first check is probably not required.

Reply:
eb308a453240d2ec0ea3665991cd5935f5113ab0